src/Management/test/Endpoint.Test/ActuatorRouteBuilderExtensionsTest.cs

```diff
@@ -46,10 +46,22 @@ static bool Query(Type t)
                 return t.GetInterfaces().Any(type => type.FullName == "Steeltoe.Management.IEndpoint");
             }
 
-            static bool SupportedOnPlatform(Type t)
+            static bool SupportedOnPlatform(Type type)
             {
-                return !(t.Name.StartsWith("ThreadDump") || t.Name.StartsWith("HeapDump")) || (t.Name.StartsWith("ThreadDump") && Platform.IsWindows) ||
-                    t.Name.StartsWith("HeapDump");
+                bool isThreadDump = type.Name.StartsWith("ThreadDump", StringComparison.Ordinal);
+                bool isHeapDump = type.Name.StartsWith("HeapDump", StringComparison.Ordinal);
+
+                if (!(isThreadDump || isHeapDump))
+                {
+                    return true;
+                }
+
+                if (isThreadDump && Platform.IsWindows)
+                {
+                    return true;
+                }
```

Comment:
```suggestion
                return isThreadDump || isHeapDump;
```

Reply:
Is that what it should be?

Reply:
Or remove `SupportedOnPlatform` entirely?

Reply:
I think we can delete this whole thing, the result should always be `true`